src/app/clients/form/formSections/ListTypeCard.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import { errorText, valueOnChangeDropdownList } from "@/components/Form/formFunctions";
+import GenericFormCard from "@/components/Form/GenericFormCard";
+import { ErrorText } from "@/components/Form/formStyling";
+import DropdownListInput from "@/components/DataInput/DropdownListInput";
+import { ClientCardProps } from "../ClientForm";
+
+type ListTypeLabelsAndValues = [string, string][];
+
+const ListTypeCard: React.FC<ClientCardProps> = ({
+    formErrors,
+    errorSetter,
+    fieldSetter,
+    fields,
+}) => {
+    const listTypeLabelsAndValues: ListTypeLabelsAndValues = [
+        ["Regular", "regular"],
+        ["Hotel", "hotel"],
+    ];
+
+    return (
+        <GenericFormCard
+            title="List Type"
+            required={true}
+            text="From which list should parcels for this client be packed using?"
```

Comment:
Happy with the actual code, solution looks good. 

Would there be a better way to word this list type question?